Tests/conf.json

```diff
@@ -219,11 +219,6 @@
             "playbookID": "Azure SecurityCenter - Test",
             "instance_names": "ms_azure_sc_prod"
         },
-        {
-            "integrations": "Azure Security Center v2",
```

Comment:
Why are there tests that are removed?

Reply:
There are three duplicated tests
I assumed this is by accident and removed them.

The locking mechanism thinks is passed a round when sees the duplication and sleeps for 30 seconds for no reason